theme/int/style.scss

```
@@ -0,0 +1,50 @@
+$primaryColor: #064963;
+$primaryColorLight: #417491;
+$primaryColorDark: #002239;
+$secondaryColor: #D9174C;
+$secondaryColorLight: #F15574;
+$secondaryColorDark: #A31139;
+$accentColor: #74C599;
+$colorDarkGrey: #17212A;
+$colorLightGrey: rgba(23, 33, 42, 0.1);
+
+$logoGradient: linear-gradient(35deg, #b1063a 35%, #f6a800);
+
+$colorBeige: #e6e2e2;
+$colorGrey: rgba(232, 232, 232, 0.5);
+
+$logoBackground: url("../../images/logo/cloud-transparent-mono.svg") no-repeat center;
+$logoBackgroundTablet: url("../../images/logo/cloud-transparent-mono-48-48.svg") no-repeat center;
+
+aside.nav-sidebar{
+	.sc-branding{
+		padding: 0 5px;
+		background: $secondaryColor;
+		.cloud-logo{
+			width: 100%;
+			margin: 0;
+			padding: 30px;
+		}
+		.sc-title {
+			display: none;
+		}
+	}
+
+	@media (max-width: 991px) {
+		.sc-branding{
+			padding: 0;
+			.cloud-logo{
+				background: $logoBackgroundTablet;
+			}
+		}
+	}
+	ul {
+		li {
+			a:not(.sc-branding) {
+				&.active {
+					color: $secondaryColor;
```

```suggestion
					color: $secondaryColorDark;
```
vgl. https://www.sketch.com/s/4d31147a-c78f-460d-a5ff-54b247a0e5cc/a/KPRa5K

---

done ;)

---

done ;)